repo/packages/K/kong/1/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "packagingVersion": "3.0",
+  "name": "kong",
+  "version": "0.10.3",
+  "scm": "https://github.com/Mashape/kong",
+  "maintainer": "https://getkong.org/community/",
```

Comment:
Since it will be supported by Mashape, it should be a valid email alias where users/customers can reach out.